### DISCUSSION TITLE
How to not scrape or suppress comment contents

### DISCUSSION THREAD

**Discussion by u3533791** [kevinzg/facebook-scraper#736] [Q&A]
I am scrapping a page for its images and captions.

When capturing `post['images']`, images / video thumbnails in comments are also captured.
Some post contains more than 1 image, so it is necessary to loop through `post['images']`.
However, I need `'allow_extra_request'` to be `True`, or I won't be able to scrape long text captions.

`"comments": False` is already specified in 
`for post in get_posts(targetpage, pages=20, options={"comments": False}):`

**Comment by neon-ninja** [comment]
What post ID are you having this problem with?

**Reply by u3533791** [reply]
Sorry for the confusion.
I tried to tweak it a bit and found the issue.
Seems the image urls are stored in post['images_lowquality'] but I used to be able to fetch them from post['images].
So, now when I tried to access post['images'], it got all buggy and stuff.
Are they substantially different? post['images_lowquality'] and post['images]?

**Reply by neon-ninja** [reply]
Yes, `images_lowquality` has low quality (low resolution) images, whereas `images` has high resolution
